examples/bootstrap-script/README.md

```diff
@@ -0,0 +1,116 @@
+# Bootstrapping your users
+
+Before spawning a notebook to the user, it could be useful to 
+do some preparation work in a bootstrapping process.
+
+One common use case is at follows:
+
+* You are using an LDAPAuthenticator and have no home directory for your user on the host
```

Comment:
Hello @yuvipanda , I didn't want to suggest to create a local user. I deleted the term "home"-directory in the description. 

Altough one major drawback of the suggested solution is that a directory once created for one user will be there forever, even when the user is kicked out of the LDAP directory. 
Maybe we should anticipate this by providing another cleanup hook once the user is deleted in the jupyterhub database?